### PR TITLE
gooddata-dbt - fix reading environment from dbt Cloud

### PR DESCRIPTION
**Pull request by jaceksan**
Switch from v2 to v3 API

Read DB_NAME from correct place

Simply the transformation from Environment to profiles.yml

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-dbt/gooddata_dbt/dbt/cloud.py [hunk 1]
```diff
@@ -249,33 +249,35 @@ def read_environment(
...
+        environment: DbtCloudEnvironment,
+        path_to_profiles: Path,
+        profile_file: str = "profiles.yml",
+    ) -> None:
         if not os.path.isdir(path_to_profiles):
             os.makedirs(path_to_profiles)
         profile_file_path = path_to_profiles / profile_file
-        data = {environment.profile: {"outputs": {environment.target: environment.to_profile()}}}
+        profile = os.getenv("ELT_ENVIRONMENT")
+        target = os.environ.get("DBT_TARGET")
```
Comment (hkad98) [inline gooddata-dbt/gooddata_dbt/dbt/cloud.py:269]: Could you use os.getenv also here so we are consistent?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR522

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/301/ : FAILURE in 29m 59s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/451/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/455/ : SUCCESS in 9s

**Pull request merged**